src/bin.ts

```diff
@@ -1,52 +1,33 @@
 #!/usr/bin/env node
 
-import { argv } from 'yargs';
```

Comment:
I just spotted smth - why we renamed half of the variables or moved their positions? it makes no difference, apart from diff size. 

Could you please keep yargs on 3rd line, please?